### DISCUSSION TITLE
v2.5.9

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#173] [Announcements]
- Fixed tab bar hiding below navigation buttons when using button navigation.

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.5.9'>v2.5.9</a>.</em>

**Comment by nimeshpatel5905** [comment]
Sir if possible can you make this for Application for TV? 

Please if you need some donations please let me know
